packages/browser/src/index.bundle.feedback.ts

```diff
@@ -2,7 +2,7 @@ import { browserTracingIntegrationShim, replayIntegrationShim } from '@sentry-in
 
 export * from './index.bundle.base';
 
-export { feedbackIntegration } from './feedback';
+export { feedbackAsyncIntegration as feedbackIntegration } from './feedbackAsync';
```

Comment:
@mydea What do you think about this alias? 

The idea is that the docs will use `feedbackAsyncIntegration` for our npm setup instructions. So that's what people will copy paste and see all over the place.

But for the CDN, most people probably won't interact with the integration name right?

I wanted to have a stable name so that we can flip from async to sync in the future if we wanted to and have it be transparent.

The CDN could alternatively ship `feedbackAsyncIntegration` to start, and then in the future ship both `feedbackAsyncIntegration` and `feedbackIntegration`.

Reply:
Actually, i'm thinking now that things like `getIntegrationByName` would still need to have `'feedbackAsyncIntegration'` passed in because that's what the instance will be called. so the alias here isn't adding confusion to that case.

Reply:
100%... I had some feeling about some of these points but the getIntegrationName i was less sure about. You laid it out well, let see what levers we can pull to improve it:

### `getIntegrationName`
This should be `Feedback` in both cases, because `buildFeedbackIntegration` returns the same object, with different dependencies injected. So this sounds like the ideal ✅ 

### CDN default impl and function name
For the CDN bundle, we'll go with the slim bundle size, and async loading strategy. So this PR will deliver that.
What's the function name though?

Some ideas:
```javascript
// basic name, no hint about strategy:
  integrations: [Sentry.feedbackIntegration()]

// strategy specific name, can help with clarifying the docs
   integrations: [Sentry.feedbackAsyncIntegration()]

// different strategy specific name:
  integrations: [Sentry.lazyFeedbackIntegration()]

// different strategy specific name (not as good):
  integrations: [Sentry.feedbackOnInteractionIntegration()]
```

I think we should write the docs and that could help a lot

Reply:
What about this:

We always (CDN & NPM) have a `feedbackIntegration`, which uses _some undefined_ loading strategy. Basically we'll use what we think works better, which on CDN is async and on NPM is sync. 

Then in addition we also have `feedbackAsyncIntegration` and `feedbackSyncIntegration`. On CDN only the former exists, on NPM both exist. 

Does that make sense?

Reply:
yes! i like this. It's good for docs-writing, because we can be consistent everywhere, and we'll just add the section for `loading strategies` to disambiguate. In that section can explain how it's Async for all the web-facing stuff, and we can make the electron package be Sync to both prove the point, but that's what I'd expect there anyway.

I'll update this PR in a sec